tests/load/load_test.go

```diff
@@ -33,15 +33,17 @@ var _ = ginkgo.Describe("[Load Simulator]", ginkgo.Ordered, func() {
 		blockchainID := subnetDetails.BlockchainID
 
 		nodeURIs := subnetDetails.ValidatorURIs
-		rpcEndpoints := make([]string, 0, len(nodeURIs))
+		nodeURIsLen := len(nodeURIs)
+
+		rpcEndpoints := make([]string, 0, nodeURIsLen)
```

Comment:
Can we revert this change? Seems unnecessary

Reply:
sure